bpf/process/string_maps.h

```diff
@@ -91,6 +91,39 @@ DEFINE_ARRAY_OF_STRING_MAPS(9)
 DEFINE_ARRAY_OF_STRING_MAPS(10)
 #endif
 
+#define POLICY_STR_OUTER_MAX_ENTRIES 1
```

Comment:
I saw some discussion here, #1408 about having one single map instead of 11 different maps. We can also accept the performance loss and use just one unique shared map to simplify things